src/main/java/org/alliancegenome/curation_api/model/entities/base/GeneratedAuditedObject.java

```diff
@@ -20,9 +20,9 @@
 import lombok.ToString;
 
 @Data
-@EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = true)
+@EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = false)
```

Comment:
@markquintontulloch this effects ALL classes. Not sure we want to set `callSuper = false`?

Reply:
Agreed - I've switched the change to the CrossReference class, which was what was causing Christian's Reference identity test to fail.